client/web/web.go

```diff
@@ -265,7 +277,7 @@ req.send(null);
 `
 
 // Handle processes all requests for the Tailscale web client.
-func Handle(w http.ResponseWriter, r *http.Request) {
+func (s *Server) Handle(w http.ResponseWriter, r *http.Request) {
```

Comment:
oh, if we're making this a method on the Server type, then we should rename it to `ServeHTTP` so then it implements the `http.Handler` interface directly.  So then callers could just do:

```go
ws := new(web.Server)
http.ListenAndServe(":8080", ws)
```

Reply:
nice okay!